jsx/App/Stories/Story/lib/txt_sync.js

```diff
@@ -1,6 +1,31 @@
 // Based on http://community.village.virginia.edu/etst/
 
 export function setupTextSync() {
+
+    // "media" is undefined if there are no AV files associated with the current text. 
+    // Holly's change on Oct 5, 2020: 
```

Comment:
This comment ("Holly's change ... console.") is useful for me now as a reviewer, but let's not merge it in. Generally, comments should be about the current code and how it works (including any hacky solutions in it), not about its history. That way, people working on this code in the future (including us!) can ignore its history and save brainpower, while people who wonder about its history can look at the version history to find that out. There's always the option to comment directly on a pull request if you think someone will want to go back and look at the rationale for things in the future.

Reply:
Updated